### PR TITLE
Fix image names for devstack jobs

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: scripts/jenkins/qa_devstack.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ function h_setup_base_repos {
         . /etc/os-release
 
         DIST_NAME=${NAME}
-        DIST_VERSION=${VERSION_ID}
+        DIST_VERSION=${VERSION}
```
Comment (mjura) [inline scripts/jenkins/qa_devstack.sh]: I think that `DIST_VERSION=${VERSION_ID}` was correct, for example for my openSUSE `VERSION="13.2 (Harlequin)"`

Comment (dirkmueller) [inline scripts/jenkins/qa_devstack.sh]: well, for SLES its wrong. so lets try a different approach (equally untested)

**Comment by mjura** [comment]
+1

**Comment by toabctl** [comment]
+1

**Pull request merged**